Gemfile

```diff
@@ -4,45 +4,23 @@ source 'https://rubygems.org'
 ruby "2.4.4"
 git_source(:github) { |repo_name| "https://github.com/#{repo_name}.git" }
 
-plugin 'bootboot', '~> 0.1.1' unless Bundler.settings[:frozen]
-Plugin.__send__(:load_plugin, 'bootboot') if Plugin.installed?('bootboot')
+gem 'rails', '~> 5.0.0'
 
-if ENV['DEPENDENCIES_NEXT']
-  enable_dual_booting if Plugin.installed?('bootboot')
-
-  # This will only be loaded when running
-  # bundler command prefixed with `DEPENDENCIES_NEXT=1
-  gem 'rails', '> 5.0', '< 5.1'
-
-  gem 'activemerchant', '>= 1.78.0'
-  gem 'angular-rails-templates', '>= 0.3.0', '< 1.1.0'
-  gem 'awesome_nested_set'
-  gem 'rails-controller-testing'
-  gem 'ransack', '2.3.0'
-  gem 'responders'
-else
-  gem 'rails', '~> 4.2'
-
-  gem 'activemerchant', '~> 1.78.0'
-  gem 'angular-rails-templates', '~> 0.3.0'
-  gem 'awesome_nested_set', '~> 3.3.1'
-  gem 'ransack', '~> 1.8.10'
-  gem 'responders', '~> 2.0'
-
-  gem 'db2fog'
-  gem 'unicorn'
-
-  group :test do
-    gem 'test_after_commit' # needed to test Devise callbacks
-  end
-end
+gem 'activemerchant', '>= 1.78.0'
+gem 'angular-rails-templates', '>= 0.3.0'
+gem 'awesome_nested_set'
+gem 'ransack', '2.3.0'
+gem 'responders'
+gem 'sass', '<= 4.7.1'
+gem 'sass-rails', '< 6.0.0'
 
 gem 'i18n'
 gem 'i18n-js', '~> 3.8.0'
 gem 'rails-i18n'
 gem 'rails_safe_tasks', '~> 1.0'
 
 gem "activerecord-import"
+gem "db2fog", github: "openfoodfoundation/db2fog", branch: "rails-5"
```

Comment:
IMO this needs a tech debt issue so we get to a solution that doesn't require us to maintain a fork.

Reply:
There were some tech-debt issues related to switching to a different gem. I tried it out, but the results were not good. We lost functionality and actually the alternative gem broke some AWS-related stuff like image paths.

Reply:
I guess we can leave an issue open to explore other options :+1:

Reply:
we might need to explore solutions then. You know what happens when forks go unmaintained...

Reply:
The only other option I could see was writing a custom database backup solution from scratch...

Reply:
this can be solved from ofn-install as well. Backing up a PostgreSQL it's nothing we can only do in Ruby.